lib/ui/onboarding.dart

```diff
@@ -45,8 +56,10 @@ class _OnboardingScreenState extends State<OnboardingScreen> {
             children: [
               pageBuilder(
                 Text('haveYouHeard...'.tr,
-                    style: HyhTextStyle.body16Height15),
+                    textScaleFactor: size.height/667,
```

Comment:
Where does 667 come from?

Reply:
it's the size of the design screen, it keeps the designed proportion